src/apps/xcalibur/arbitrum/xcalibur.pool.token-fetcher.ts

```diff
@@ -0,0 +1,102 @@
+import { Inject } from '@nestjs/common';
+import { BigNumber, BigNumberish } from 'ethers';
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
+import { Erc20 } from '~contract/contracts';
+import { AppTokenTemplatePositionFetcher } from '~position/template/app-token.template.position-fetcher';
+import {
+  GetAddressesParams,
+  DefaultAppTokenDefinition,
+  GetUnderlyingTokensParams,
+  UnderlyingTokenDefinition,
+  GetPricePerShareParams,
+  DefaultAppTokenDataProps,
+} from '~position/template/app-token.template.types';
+
+import { XcaliburContractFactory, XcaliburSwapFactory, XcaliburRouter, UniswapPairV2, XcaliburToken} from '../contracts';
+
+@PositionTemplate()
+export class ArbitrumXcaliburPoolTokenFetcher extends AppTokenTemplatePositionFetcher<XcaliburToken> {
+  groupLabel: string = 'Your Group Label';
```

Comment:
You want to groupLabel which represent the type of position. In this case, I suggest `Pools`